apps/dashboard/app/views/batch_connect/session_contexts/_form.html.erb

```diff
@@ -6,12 +6,10 @@
     <%= create_widget(f, attrib, format: @render_format) %>
   <% end %>
 
-  <div class="form-group">
-    <div class="form-check">
-      <input class="form-check-input" name="save_template" type="checkbox" id="batch_connect_session_save_template">
-      <label class="form-check-label" for="batch_connect_session_save_template"><%= t('dashboard.batch_connect_form_save') %></label>
-      <input name="template_name" type="text" id="batch_connect_session_template_name" readonly>
-    </div>
+  <div class="form-group form-check d-flex">
+      <input class="form-check-input align-self-center" name="save_template" type="checkbox" id="batch_connect_session_save_template">
+      <label class="form-check-label align-self-center" for="batch_connect_session_save_template"><%= t('dashboard.batch_connect_form_save') %></label>
+      <input class="ml-auto" name="template_name" type="text" id="batch_connect_session_template_name" aria-label="<%= t('dashboard.batch_connect_form_template_name_label') %>" readonly>
```

Comment:
These are cosmetic changes to align these items.  Plus it gives an `aria-label` to the readonly input because the screen reader has no label for the same (visual users don't have a label either - but proximity should indicate what it is).

Before:
![image](https://github.com/OSC/ondemand/assets/4874123/a461de42-1d09-4a80-b362-bfdca99a0e9c)



After:
![image](https://github.com/OSC/ondemand/assets/4874123/b43e50b4-32be-4c86-ad9c-76c993198c3b)